site/layout_scales.js

```diff
@@ -0,0 +1,123 @@
+import React from 'react';
+import fs from 'fs';
+import path from 'path';
+import execall from 'execall';
+
+const layoutCss = fs.readFileSync(path.join(__dirname, '../src/layout.css'), 'utf8');
+
+// Expects regular expressions with one sub-match that is the number
+function findNumbers(re) {
+  return execall(re, layoutCss).map((match) => {
+    return match.sub[0];
+  });
+}
+
+const classSets = {
+  'Grid Gutters': {
+    classPatterns: ['grid--gut{n}'],
+    numbers: findNumbers(/\.grid--gut(\d+) {/g)
+  },
+  'Z-Indexes': {
```

Comment:
I'm feeling like we could remove z-indexes from this page— the pattern is much more memorable than the other scales being documented on this page, and z-indexes are the only non-pixel unit?